qemu/tests/win_heavyload.py

```diff
@@ -139,9 +139,18 @@ def add_option(cmd, key, val):
             if not installed and download_url:
                 utils.system("rm -f %s/HeavyLoad*.exe" % tmp_dir)
                 session.cmd("del /f /s %sHeavyLoad*.exe" % dst)
-            if loop_session_cmd(session, check_running_cmd):
-                if not loop_session_cmd(session, stop_cmd):
-                    raise error.TestFail("Unable to terminate heavyload "
-                                         "process")
+            i = 0
```

Comment:
The function loop_session_cmd is useless now, need remove the function definition.